### ISSUE TITLE
Maps are blank when previewing in RStudio on Windows with R 4.4.0

### ISSUE THREAD

**Issue by AMBarbosa** [rstudio/leaflet#918]
This was working fine with my previous version of R (4.3.2) on RStudio, but after updating **R** to version **4.4.0** on Windows (haven't updated R on Linux yet), I'm getting a blank map viewer every time I try a leaflet map (including the `?leaflet` Examples), both with the previous and with the newest version of RStudio (released 2024-04-29). It's working OK on R without RStudio, though.

**Comment by Bevann** [comment]
I am also running into this issue.  Mapview and tmap operations that rely on leafltet are also not rendering in viewer, or when displayed in browser.

**Comment by gadenbuie** [comment]
There is a bug with R 4.4.0 on Windows that affects RStudio's preview of htmlwidgets (and other HTML content). Please follow the issue here for more information: https://github.com/rstudio/rstudio/issues/14603

**Comment by AndrewMcMillanLandcare** [comment]
Solution to problem:

I have just installed R Version 4.4.0 and upgraded Rstudio to v 2024.04.0 build 735. I was suddenly unable to use MapView with the same symptoms as described above. The Viewer pane just came up blank. This was also true for trying to display the widget produced by ggplotly in the plotly package.

Just by fluke I discovered by putting the command 

`options(viewer = NULL) `

before the mapview command solved the issue.

For the example above, try:

    library(mapview)
    options(viewer = NULL)
    mapview(breweries)
